network_insight_sdk_generic_datasources/routers_and_switches/cisco/cisco_pre_post_processor.py

```diff
@@ -3,9 +3,30 @@
 
 import re
 
+from network_insight_sdk_generic_datasources.common.log import py_logger
 from network_insight_sdk_generic_datasources.common.utilities import merge_dictionaries
 from network_insight_sdk_generic_datasources.parsers.text.pre_post_processor import PrePostProcessor
 from network_insight_sdk_generic_datasources.parsers.common.block_parser import SimpleBlockParser
+from network_insight_sdk_generic_datasources.parsers.text.text_processor import rule_match_callback
+from network_insight_sdk_generic_datasources.parsers.text.text_processor import Rule
+from network_insight_sdk_generic_datasources.parsers.text.text_processor import TextProcessor
+from network_insight_sdk_generic_datasources.parsers.common.block_parser import LineBasedBlockParser
+
+NAME_KEY = 'name'
+MTU_KEY = 'mtu'
+IP_KEY = 'ipAddress'
+IF_SPEED_KEY = 'interfaceSpeed'
+ADMIN_ST_KEY = 'administrativeStatus'
+OP_ST_KEY = 'operationalStatus'
+HW_KEY = 'hardwareAddress'
+DUPLEX_KEY = 'duplex'
+OP_SPEED_KEY = 'operationalSpeed'
+ACTIVE_PORTS_KEY = 'activePorts'
+PASSIVE_PORTS_KEY = 'passivePorts'
+VLAN_KEY = 'vlan'
+CONNECTED_KEY = 'connected'
+SW_PORT_KEY = 'switchPortMode'
```

Comment:
Can we move all these keys to common place ? Then we can reuse. Check if there is a constant python file.

Reply:
Done.